fvtest/compilertriltest/CMakeLists.txt

```diff
@@ -39,6 +39,7 @@ add_executable(comptest
 	ShiftAndRotateTest.cpp
 	SimplifierFoldAndTest.cpp
 	IfxcmpgeReductionTest.cpp
+    VectorTest.cpp
```

Comment:
Fix formatting, unless there's a reason why this is not aligned with the rest?

Reply:
Fixed. Some spaces snuck in.